src/main/java/com/szymon/controller/BookController.java

```diff
@@ -0,0 +1,72 @@
+package com.szymon.controller;
+
+import com.szymon.domain.Book;
+import com.szymon.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * Created by szymon.nowak on 04.04.2016.
+ */
+@RestController
+@RequestMapping(value = "/books")
+public class BookController {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET)
+    public Book get(@PathVariable("id") int id) {
+        return bookRepository.findOne(id);
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public List<Book> listAll() {
+        return bookRepository.listAll();
+    }
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
+    public Book delete(@PathVariable("id") int id) {
+        return bookRepository.delete(id);
+    }
+
+
+    // Method to POST some default books. Can be used for testing.
+    @RequestMapping(value = "/addBasicBooks")
+    public List<Book> addBasicBooks() {
```

Comment:
This look like init method. Please read about init methods for services in Spring.